pkg/provisioner/controller.go

```diff
@@ -53,6 +54,8 @@ type Controller struct {
 	obHasSynced  cache.InformerSynced
 	queue        workqueue.RateLimitingInterface
 
+	provisionerLabels map[string]string
```

Comment:
can you add a comment saying these labels are optional and defined completely by each provisioner?

Reply:
nit: not sure if there's consistency with the use of blank lines in this struct. I'd prefer a comment before each field and no blank lines.